gensim/matutils.py

```diff
@@ -479,6 +479,38 @@ def isbow(vec):
         return False
     return True
 
+def symmetric_kl(distrib_p, distrib_q):
+    return numpy.sum([stats.entropy(distrib_p, distrib_q), stats.entropy(distrib_p, distrib_q)])
```

Comment:
I was thinking that ``numpy.sum`` could be faster than ``+`` for the NumPy matrix operation.